lmatools/flashsort/flash_stats.py

```diff
@@ -179,8 +179,11 @@ def calculate_flash_stats(flash, min_pts=2):
             x[0] += perturb[0]
             y[0] += perturb[1]
             z[0] += perturb[2]
-            volume, vertices, simplex_volumes = hull_volume(np.vstack((x,y,z)).T)
-    
+            try:
+                volume, vertices, simplex_volumes = hull_volume(np.vstack((x,y,z)).T)
+            except Exception as e:
+                pass
```

Comment:
I assume this still runs since volume is set to zero above, and vertices and simplex_volumes aren't needed?

An alternate fix might be to require >4 instead of >3 points, as [I recently did in xlma-python](https://github.com/deeplycloudy/xlma-python/commit/4cf8f70eecd3afd0a102a98ef5b4175994803949).

Reply:
I think I answered this in #38, feel free to accept whichever you want

Reply:
Per the answer in #38, it sounds like adding one more point (to area and volume) fixed the problem. Please update this PR to require one more point, and remove the try/except. With that, I think this PR will be ready to go!